.github/actions/node/action.yml

```diff
@@ -0,0 +1,34 @@
+name: 'Setup Node Workspace'
```

Comment:
nit: rename file to `setup.yml`?

Reply:
can't, github requires an action to be named as `action.yml`